ginivision/src/main/java/com/ortiz/touch/TouchImageView.java

```diff
@@ -75,8 +74,7 @@
     private Fling fling;
     
     private ScaleType mScaleType;
-    
-    private boolean imageRenderedAtLeastOnce;

```

Comment:
Removed this because setting an image at a later time (after loading the file and creating a bitmap) did not work.